Weekend push

msdickinson opened the pull request:
See Issue #125, If you have a concern with
"import { default as NQB } from '../../game/sidebar/next-quarter-button/next-quarter-button.jsx';"
In main-side-bar-.jsx file

Commit:
Made Changes To Show Main Page.

Commit:
Put In throw just before game. Plz look at branch and look at game. Before
we meet (via internet phone)

Commit:
Adding In Crash

Commit:
Fixed Bug Related To UI

Commit:
Merge branch 'develop' of https://github.com/gios-asu/dont-get-fired-game into WeekendPUSH
Fixed UI Components. Game Page now renders correctly.

Commit:
Created New BackEnd Files Called CS-... Forming them to the pages, and
going to warp the events around them. You can see programs, and buy them.
Game is more functional.

rraub commented:
FYI your MainSideBar folder should be renamed main-side-bar

rraub commented:
Also your git email isn't setup right on your client, which is why all the commits are not attributed to you. 
`git config --global user.email "your_email@example.com"`

msdickinson commented:
thank you, will fix tonight, and have it ready to be pulled before morning. so you can will have it first thing tomorrow morning.

xyan12345 committed:
add cs-program data

xyan12345 committed:
modify program data

xyan12345 committed:
Modified Program Json

xyan12345 committed:
Modified Data Program

drsmall committed:
Added randomization for game settins

Commit:
Broke 10 Tests. But Nearly all parts work.
Advisors Logic needs fine tuned,
Final Page Needs Fine Tuned
This will be our Thusday Starting Place. 7 Days Left, so pushing anyway.

xyan12345 committed:
Fixed part of issue...

xyan12345 committed:
Fix issue

rraub committed:
Merge pull request #128 from gios-asu/xuefeng-develop

Xuefeng develop

drsmall committed:
Fix linting issues

drsmall committed:
Merge branch 'WeekendPUSH' of http://github.com/gios-asu/dont-get-fired-game into david-capstone-develop

rraub committed:
Merge pull request #129 from gios-asu/david-capstone-develop

Added randomization for game settins

xyan12345 committed:
Fix program data

rraub committed:
Merge pull request #132 from gios-asu/xuefeng-develop

Fix program data within range

Commit:
Tied Dans Data Into Setup, Cleaned up setup. Some code was messed up
during a merdge Fixed. Game now uses programs for the data folder.

Commit:
Added Events Page To Flow, Setup Random Events, And Processed The Event
option that was picked. We will need alot of data for cs-events.

Commit:
Updated Advisors To Look at Goals. If they are hiting there goal/
currentQuarter/totalQuarters) It shows GOOD. If your currently hitting
half of your goal given your current quater shows OKAY, if your doing less
then that shows BAD. Static Messages could be improved though

xyan12345 committed:
cs-event data Changed

xyan12345 committed:
Merge pull request #133 from gios-asu/xuefeng-develop

cs-event data Changed

Commit:
Fixed Lint errors
